alpha/alpha-proc/src/main/java/org/neo4j/graphalgo/centrality/ParticleFilteringProc.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) 2017-2020 "Neo4j,"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.graphalgo.centrality;
+
+import org.neo4j.graphalgo.core.CypherMapWrapper;
+import org.neo4j.graphalgo.results.CentralityScore;
+import org.neo4j.graphdb.Direction;
+import org.neo4j.graphdb.Node;
+import org.neo4j.graphdb.Relationship;
+import org.neo4j.graphdb.RelationshipType;
+import org.neo4j.kernel.api.KernelTransaction;
+import org.neo4j.logging.Log;
+import org.neo4j.procedure.*;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
+
+public class ParticleFilteringProc {
```

Comment:
The Javadoc of the class should contain the full reference:

> Gallo, Denis; Lissandrini, Matteo; and Velegrakis, Yannis.
“Personalized Page Rank on Knowledge Graphs: Particle Filtering is all you need!.”
Proceedings of the 23th International Conference on Extending Database Technology, EDBT 2020 (169-180). 
http://dx.doi.org/10.5441/002/edbt.2020.54